### PR TITLE
Usercode support for totalconnect

### PR DESCRIPTION
**Pull request by austinmroczek**
## Proposed change

Adds documentation about "usercodes" to support some users that require this entry in addition to username and password in order to arm or disarm their systems.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/39199
- Link to parent pull request in the Brands repository: n/a
- This PR fixes or closes issue: n/a

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
There are unrelated commits in this PR. Please git rebase your PR onto the `next` branch, as it seems like it was branched of from the `current` branch originally.

Furthermore, please make sure the title of the PR is descriptive. "Patch 10" is not something that rings a bell when browsing through the list of PRs.

Thanks! 👍

**Comment by austinmroczek** [comment]
Updated title and re-based

**Comment by austinmroczek** [comment]
I need to update this now that YAML support is removed in the core PR, but I've been fighting with my dev setup so not quite ready yet.

File: source/_integrations/totalconnect.markdown [hunk 1]
```diff
@@ -11,20 +11,12 @@ ha_codeowners:
...
 ---
 
-The `totalconnect` integration provides connectivity with the Honeywell TotalConnect alarm systems used by many alarm companies.  
+The `totalconnect` integration provides connectivity with TotalConnect alarm systems used by many alarm companies.  
 
 ## Configuration
 
 To enable TotalConnect via the user interface, go to **Configuration** > **Integrations** > the "plus" button > Total Connect.
 
-To enable TotalConnect via `configuration.yaml` add the following lines:
```
Comment (frenck) [inline source/_integrations/totalconnect.markdown]: I'm confused, we remove YAML config, but we leave in the YAML config block down below?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/totalconnect.markdown (hunk 1 above)
Comment (austinmroczek) [reply source/_integrations/totalconnect.markdown]: It's removed now.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by MartinHjelmare** [comment]
There's a merge conflict. Please rebase on latest next branch.

**Review by frenck — APPROVED** [review]
Resolved the conflict. ✅

**Pull request merged**